Mathlib/Dynamics/TopologicalEntropy/NetEntropy.lean

```diff
@@ -0,0 +1,408 @@
+/-
+Copyright (c) 2024 Damien Thomine. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damien Thomine, Pietro Monticone
+-/
+import Mathlib.Dynamics.TopologicalEntropy.CoverEntropy
+
+/-!
+# Topological entropy via nets
+We implement Bowen-Dinaburg's definitions of the topological entropy, via nets.
+
+All is stated in the vocabulary of uniform spaces. For compact spaces, the uniform structure
```

Comment:
I don't think it's worth having the same text as in `CoverEntropy`. Maybe just give a pointer? What would be more useful is describing what the definition via nets is, for people who have never seen topological entropy.